src/extractAndWritePOTFromMessagesSync.js

```diff
@@ -12,10 +13,17 @@ const customKeyMapper = (message, messageKey, filename) => ({
 const customKeyMapperFactory = (messageKey = 'defaultMessage') =>
   (message, filename) => customKeyMapper(message, messageKey, filename);
 
-function extractAndWritePOTFromMessagesSync(srcPatterns, { messageKey, output }) {
+function extractAndWritePOTFromMessagesSync(srcPatterns, { messageKey, output, headerOptions }) {
```

Comment:
Awesome! 
It will be nice if we could mention the headerOptions argument in README file. 😀